docs/changelog_3_4_0.rst

```diff
@@ -0,0 +1,145 @@
+.. 3.4.x Changelogs
+
+Redbot 3.4.0 (Unreleased)
+=========================
+
+| Thanks to all these amazing people that contributed to this release:
+| :ghuser:`Dav-Git`, :ghuser:`DevilXD`, :ghuser:`Drapersniper`, :ghuser:`flaree`, :ghuser:`jack1142`, :ghuser:`kablekompany`, :ghuser:`Kowlin`, :ghuser:`maxbooiii`, :ghuser:`mikeshardmind`, :ghuser:`NeuroAssassin`, :ghuser:`PredaaA`, :ghuser:`Predeactor`, :ghuser:`retke`, :ghuser:`SharkyTheKing`, :ghuser:`thisisjvgrace`, :ghuser:`Tinonb`, :ghuser:`TrustyJAID`, :ghuser:`Twentysix26`, :ghuser:`Vexed01`
+|
+| **Read before updating**:
+| 1. Red 3.4 comes with support for data deletion requests. Bot owners should read `red_core_data_statement` to ensure they know what information about their user bot stores.
```

Comment:
```suggestion
| 1. Red 3.4 comes with support for data deletion requests. Bot owners should read `red_core_data_statement` to ensure they know what information about their users is stored by the bot.
```

Reply:
Since this is already highlighted ... `red_core_data_statement` should be hyperlinked to `red_core_data_statement` doc page